msmbuilder/tpt/committor.py

```diff
@@ -77,7 +77,10 @@ def committors(sources, sinks, msm):
     sinks = np.array(sinks, dtype=int).reshape((-1, 1))
 
     n_states = msm.n_states_
-    tprob = msm.transmat_
+    if hasattr(msm, 'all_transmats_'):
+        tprob = msm.all_transmats_.mean(0)
```

Comment:
is this `axis=0`? Can you add the keyword so readers of the code won't be confused